internal/cli/atlas/privateendpoints/describe.go

```diff
@@ -75,5 +75,7 @@ func DescribeBuilder() *cobra.Command {
 	cmd.Flags().StringVar(&opts.ProjectID, flag.ProjectID, "", usage.ProjectID)
 	cmd.Flags().StringVarP(&opts.Output, flag.Output, flag.OutputShort, "", usage.FormatOut)
 
+	cmd.Deprecated = "Please use mongocli atlas privateEndpoint(s)|privateendpoint(s) aws|azure describe <ID> [--projectId projectId]"
```

Comment:
💯 but let's not show the alias here
```suggestion
	cmd.Deprecated = "Please use mongocli atlas privateEndpoints aws describe <ID> [--projectId projectId]"
```
You can also omit azure as the original command was aws only